Pre-processing/PlotOneChannel.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from mne.io import concatenate_raws, read_raw_edf
+
+
+#Load Data
+load_path = 'C:/Users/User/Desktop/CSE 481C - Neruo Capstone/Project Repo/Data/TUH/00003010_s003_t000.edf'
+print(load_path)
+raw = read_raw_edf(load_path, preload=True)
+
+#Inspect the .edf file
+print(raw.info['ch_names'])
```

Comment:
do we need to inspect the file when running? Or are these just for debugging? If the latter, I think we can remove all inspection print statements.